Changelog for 2.0.6 (#174)
twilio/twilio-voice.js: CHANGELOG.md

2.6.0 (June 20, 2023)
=====================

Changes
-------

- The SDK now builds on NodeJS versions 16 and above without the `--legacy-peer-deps` flag.
- Removed usage of NodeJS modules from the SDK and some dependencies. With this change, the SDK should now work with some of the latest frameworks that use the latest versions of bundlers such as Vite and Webpack.

> - Migrated the `AudioPlayer` dependency and is now part of the SDK. This change fixes an issue where source maps are not properly loaded.

Reviewer: ```suggestion
- Migrated the `AudioPlayer` dependency into the SDK. This change fixes an issue where source maps are not properly loaded.
```

> - Migrated the `AudioPlayer` dependency and is now part of the SDK. This change fixes an issue where source maps are not properly loaded.

Reviewer 2: The `AudioPlayer` dependency has been incorporated into the SDK as part of a migration. This change fixes an issue where source maps are not properly loaded.
